### DISCUSSION TITLE
[Feature] Backup external storage Android

### DISCUSSION THREAD

**Discussion by vladwing** [immich-app/immich#9162] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would like to be able to use my Android phone to sync/upload photos from my DSLR camera. I have a Nikon D7000 which is only visible as a MTP device, which I understand it might be tedious to support.

I noticed that the Android app doesn't recognize external storage - ie. SD cards connected to the phone. Would that be difficult to support/add?

Is there any other way you suggest for this type of usage?

PS. I checked to the best of my abilities for this feature/request. Because of the overloaded terms, I couldn't find anything, sorry if it's a duplicate.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Is this a particularly special case, or does it just present as a folder on the phone's filesystem? If so it might be a duplicate of #7075

**Reply by vladwing** [reply]
On Android it should be visible as a different "Storage device", but Immich Android doesn't see that folder or the external storage.
![Storage_devices](https://github.com/immich-app/immich/assets/2703642/95e6a26b-2575-407c-af1d-a6ae24f81fce)

**Comment by shawnbonifas** [comment]
I found this because I was looking to see if the SD card would be accessed. I think because the app is using the photos and videos permission it sees both the internal and external storage, and the folders are not labeled as such. For instance, I have two camera folders available when selecting folders to backup. One is on the SD card and the other is internal.

**Comment by notAnElephant** [comment]
has this been addressed yet?

**Comment by JakeMZTF** [comment]
I'm also stuck on no SD card folders showing in app or syncing. Pretty big hole in the app as implemented.
